go/cmd/nomos-wait/main_test.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import "testing"
+
+func TestParse(t *testing.T) {
+	testData := `Current   Context    Status           Last Synced Token   Sync Branch
+-------   -------    ------           -----------------   -----------
+*         kf-ci-v1   SYNCED           79629ca7            gcp_blueprints`
+
+	status, commit, err := parseStatus(testData)
+	//#err := parseS(testData)
+
+	if err != nil {
+		t.Errorf("Expected nil; got error %v", err)
+	}
+
+	expected := "79629ca7"
+	if commit != expected {
+		t.Errorf("Got commit: %v; want %v", commit, expected)
```

Comment:
nit/question: could [`EqualErrorf`](https://github.com/stretchr/testify/blob/51595dcf940cfb09eb91b6b78babba78bb2bfb25/assert/assertion_format.go#L86) be used instead?